style.css

```diff
@@ -0,0 +1,3 @@
+.clearfix:before,.clearfix:after{content:"";display:table}.clearfix:after{clear:both}
+
+/*# sourceMappingURL=style.css.map */
```

Comment:
I'm not sure how to add a blank line as this file is generated (other than manually doing it each time). I don't see options for it in the `grunt-sass` task. Ideas?